packages/loopring_v3/contracts/aux/access/LoopringIOExchangeOwner.sol

```diff
@@ -42,15 +42,20 @@ contract LoopringIOExchangeOwner is SelectorBasedAccessManager
             hasAccessTo(msg.sender, SUBMITBLOCKS_SELECTOR) || open,
             "PERMISSION_DENIED"
         );
-        bytes memory decompressed = LzDecompressor.decompress(data);
+        bytes memory decompressed = ZeroDecompressor.decompress(data);
         require(
             decompressed.toBytes4(0) == SUBMITBLOCKS_SELECTOR,
             "INVALID_DATA"
         );
 
-        (bool success, bytes memory returnData) = target.call(decompressed);
-        if (!success) {
-            assembly { revert(add(returnData, 32), mload(returnData)) }
+        address addr = target;
+        assembly {
+            let success := call(gas(), addr, 0, add(decompressed, 32), mload(decompressed), decompressed, 0)
+            if eq(success, 0) {
```

Comment:
> Should we change to use eq in other places? I think we have quite a few cases...

> It seems to handle the return as well. In our case, we don't return any data, or should we ?

Shouldn't matter, just using eq here because there is no return value for `submitBlocks` and to keep everything in assembly.

> We also have this pattern:

Testing has shown this needs additional memory copies, the new method doesn't.